main.py

```
@@ -0,0 +1,146 @@
+# 특정 회원 조회
+class SearchData(BaseModel):
+    nickname: str = Field(min_length=4, max_length=8)
+    profile_image: Optional[str] = None
+    created_at: datetime
+
+class SearchResponse(BaseModel):
+    status: str
+    data: SearchData
+
+@app.get("/users/{nickname}", response_model=SearchResponse)
+async def get_profile(
```

Ruff (0.14.11)

Redefinition of unused `get_profile` from line 76: `get_profile` redefined here

(F811)